Automattic/wp-calypso
# Full Site Editing: remove switch to draft while editing templates

#### Changes proposed in this Pull Request

Same as for trashing, we don't want to allow moving templates to drafts
since this might negatively impact editing and rendering functionality.

| Before | After |
| - | - |
| <img width="305" alt="Screenshot 2019-08-09 at 16 20 40" src="https://user-images.githubusercontent.com/1182160/62785808-a6b75d00-bac1-11e9-8454-b0af92000849.png"> | <img width="274" alt="Screenshot 2019-08-09 at 16 21 43" src="https://user-images.githubusercontent.com/1182160/62785888-cd759380-bac1-11e9-975e-aabb113ed5dd.png"> |


#### Testing instructions

1. Open header or footer for editing on your FSE test site.
2. Verify that `Switch to Draft` button is no longer visible.
